documentation/specs/src/base-ledger/core-concepts.md

```diff
@@ -0,0 +1,15 @@
+# Consensus
+
+Namada uses [Tendermint Go](https://github.com/tendermint/tendermint) through the [tendermint-rs](https://github.com/heliaxdev/tendermint-rs) bindings in order to provide peer-to-peer transaction gossip, BFT consensus, and state machine replication for Namada's custom state machine.
+
+## Default account
```

Comment:
default account, fungible token, and k-of-n multisig should still be in separate files since we need to flesh our their specs fully